airbyte/caches/_state_backend.py

```
@@ -20,13 +20,12 @@
     StateBackendBase,
 )
 from airbyte.exceptions import PyAirbyteInputError, PyAirbyteInternalError
+from airbyte.shared.sql_processor import SqlConfig
```

Ruff (0.11.9)

Move application import `airbyte.shared.sql_processor.SqlConfig` into a type-checking block

Move into type-checking block

(TC001)